src/components/priceDetails/PriceDetails.jsx

```diff
@@ -0,0 +1,86 @@
+import "./PriceDetails.scss";
+import React, { useEffect, useState } from "react";
+import { Button } from "@patternfly/react-core";
+import { Link } from "react-router-dom";
+import ApplyCoupon from "../../assets/icons/ApplyCoupon";
+import { useSelector } from "react-redux";
+
+
+
+function PriceDetails({btnvalue}) {
+  const [price , setPrice] = useState();
+  const { cartItems } = useSelector((state) => state.cart);
+
+  const totalAmount = () =>{
+    let total = 0;
+    cartItems.forEach((item) => {
+      total += item.id * item.quantity;
+    })
+    setPrice(total);
+  }
+
+  const getRoute = (btnvalue) => {
+    switch(btnvalue){
+      case "Checkout":
+        return "/address";
+      case "checkout":
+        return "/personalize-your-cake";
+      case "payment":
+        return "/payment";
+      default:
+        return "";
+    }
+  }
+
+  const route = getRoute(btnvalue);
+
+  useEffect(()=>{
+    totalAmount();
+  }, [cartItems]);
+
+  return (
+    <div className="price-details-main-container">
+      <div className="price-detials-apply-coupon-contianer">
+        <ApplyCoupon />
+        <button className="apply-coupon-heading">Apply coupon</button>
+      </div>
+      <div className="price-details-info-container">
+        <p className="price-details-heading">Price Details</p>
```

Comment:
Can we use https://www.patternfly.org/components/text pattternfly text component here for all h1 to h6 and p tag ?

Reply:
replaced h1 to h6 and all other tags with patternfly text components